packages/suite/src/hooks/wallet/useSendForm.ts

```diff
@@ -227,10 +227,12 @@ export const useSendForm = (props: UseSendFormProps): SendContextValues => {
         // keepDefaultValues will set `isDirty` flag to true
         reset(values, { keepDefaultValues: true });
         setLoading(false);
-        const valid = await trigger();
-        if (valid) {
-            composeRequest();
-        }
+        validateImportedTransaction(async () => {
```

Comment:
😬 It works but... I wish there was a simpler way.

Reply:
#metoo